### PR TITLE
Set config file for systemd nrpe.service

### PR DESCRIPTION
**Pull request by scopenco**
Current version of cookbook does not support change of user/group in OSs that use systemd as default init system, for example CentOS 7.
This patch adds needed functionality.

### CODE REVIEW

**Comment by scopenco** [comment]
any updates?

File: recipes/configure.rb [hunk 1]
```diff
@@ -67,6 +67,20 @@
   notifies :restart, "service[#{node['nrpe']['service_name']}]"
 end
 
+if (platform?('fedora') && node['platform_version'].to_i >= 20) || (platform_family?('rhel') && node['platform_version'].to_i == 7)
+  template "/usr/lib/systemd/system/nrpe.service" do
```
Comment (snkutlu) [inline recipes/configure.rb]: Travis-CI checks fails (in fact rubocop check) on this line for double quote usage. Can you please change them to single-quote..

**Comment by snkutlu** [comment]
Hi @scopenco 

One additional step is required for systemd which is to reload daemon with "systemctl daemon-reload" command if a new systemd service is added / changed / etc. Can you please include this as well.

Can you also think of Ubuntu and Debian latest versions as they use systemd as well?

Also for happy Travis-CI, can you take into account the inline comment I made in your PR?

Thanks
-Suleyman

File: recipes/configure.rb [hunk 2]
```diff
@@ -67,6 +67,20 @@
   notifies :restart, "service[#{node['nrpe']['service_name']}]"
 end
 
+if (platform?('fedora') && node['platform_version'].to_i >= 20) || (platform_family?('rhel') && node['platform_version'].to_i == 7)
```
Comment (sspans) [inline recipes/configure.rb]: I think using "platform" and "platform_family" will trigger warnings on chef-12.
How about replacing this line with:

```
if node['init_package'] == "systemd"
```

**Comment by snkutlu** [comment]
@scopenco If you are happy with PR #49 can you close this one?

TIA
-SNK

**Comment by scopenco** [comment]
ok

**Pull request closed without merging**